packages/flutter/test/material/page_transitions_theme_test.dart

```diff
@@ -519,4 +524,108 @@ void main() {
     expect(find.text('push'), findsOneWidget);
     expect(find.text('page b'), findsNothing);
   }, variant: TargetPlatformVariant.all());
+
+  testWidgets('ZoomPageTransitionsBuilder uses theme canvasColor during transition effects', (WidgetTester tester) async {
+    final Map<String, WidgetBuilder> routes = <String, WidgetBuilder>{
+      '/': (BuildContext context) => RepaintBoundary(
+        child: Scaffold(
+          appBar: AppBar(
+            title: const Text('Home Page'),
+          ),
+          body: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                ElevatedButton(
+                  onPressed: () {
+                    Navigator.pushNamed(context, '/scaffolded');
+                  },
+                  child: const Text('Route with scaffold!'),
+                ),
+                ElevatedButton(
+                  onPressed: () {
+                    Navigator.pushNamed(context, '/not-scaffolded');
+                  },
+                  child: const Text('Route with NO scaffold!'),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+      '/scaffolded': (BuildContext context) => RepaintBoundary(
+        child: Scaffold(
+          appBar: AppBar(
+            title: const Text('Scaffolded Page'),
+          ),
+          body: Center(
+            child: ElevatedButton(
+              onPressed: () {
+                Navigator.pop(context);
+              },
+              child: const Text('Back to home route...'),
+            ),
+          ),
+        ),
+      ),
+      '/not-scaffolded': (BuildContext context) => RepaintBoundary(
+        child: Center(
+          child: ElevatedButton(
+            onPressed: () {
+              Navigator.pop(context);
+            },
+            child: const Text('Back to home route...'),
+          ),
+        ),
+      ),
+    };
+
+    await tester.pumpWidget(
+      RepaintBoundary(
+        child: MaterialApp(
+          theme: ThemeData(
+            canvasColor: const Color.fromARGB(255, 255, 0, 0),
+            primarySwatch: Colors.blue,
+          ),
+          routes: routes,
+        ),
+      ),
+    );
+
+    // Verify initial page
```

Comment:
Nit: All comments should end in a period.

Reply:
I was under the impression that comments ending with a period were only for public facing documentation, I even looked at the other comments in the file and saw some without. Can you confirm they are indeed supposed to use periods?

Reply:
I should have mentioned it's an uber nit. It's especially important in public documentation, but should probably also be used for non-public comments too: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#leave-breadcrumbs-in-the-comments, https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar

Although I would agree there isn't a style guide example for regular comments per se.

Reply:
Sounds good, will change 👍